### DISCUSSION TITLE
I need help to quantize video-llava model to quant size 4 or 8

### DISCUSSION THREAD

**Discussion by dnabanita7** [ggml-org/llama.cpp#6029] [General]
I am trying to convert video-llava according to instructions at #2948 but I am having issues because of `convert.py` file. Since, this is llava + vicuna should I add anything extra in the convert.py file or let it remain the same?

**Comment by Abusheik008** [comment]
you can use bitsandbytes to convert the model, i assume it is an easiest way load the model from transformer
